app/views/NewMessageView.tsx

```diff
@@ -277,10 +300,8 @@ class NewMessageView extends React.Component {
 				name={item.search ? item.name : item.fname}
 				username={item.search ? item.username : item.name}
 				onPress={() => this.goRoom(item)}
-				baseUrl={baseUrl}
```

Comment:
We aren't using anymore the baseUrl and the user. So I deleted both params from this component and from mapStateToProps too.